api/docs/http_certificate_revocation_list.md

```diff
@@ -5,7 +5,11 @@ The `certificate_revocation_list` endpoint retrieves a Certificate Revocation Li
 from the master.  The master must be configured to be a CA.  The returned
 CRL is always in the `.pem` format.
 
-In all requests the `:environment` and `:nodename` must be given, but neither has any bearing on the request.
+Under Puppet Server's CA service, the `environment` parameter is ignored and can
+be omitted. Under a Rack or WEBrick Puppet master, `environment` is required and
+must be a valid environment, but it has no effect on the response.
+
+The `:nodename` is required, but has no effect on the response.
```

Comment:
I thought you had to specify `ca` as the  nodename, since you're getting the CRL associated with the CA.